queries.sql

```diff
@@ -8,3 +8,36 @@ SELECT  name, escape_attempts from animals WHERE   weight_kg > 10.5;
 SELECT * from animals WHERE   neutered = true;
 SELECT * from animals WHERE   name <> 'Gabumon';
 SELECT * from animals WHERE  weight_kg >= 10.4 and weight_kg  <= 17.3;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species is null;
+UPDATE animals SET species = 'pokemon' WHERE species is null;
```

Comment:
- [ ] This highlighted code block should be wrapped in a Transaction block. Showing the before and after like so 👇🏾 

```sql
 SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```
In the code above is just an example pointing to the before and after execution verification. But you will begin with the `BEGIN` keyword, write your query code, then verify before `COMMIT` and after.

Reply:
I have misinterpreted the instructions, I thought only the main quarries were expected to be there without transaction. But I am glad you give me such progressive feedback! 💯 @bobb-Rob